### ISSUE TITLE
Change plugin name in side menu of kibana

### ISSUE THREAD

**Issue by rohitsawant02** [opendistro-for-elasticsearch/kibana-reports#374]
Hi ,

I need to change the plugin name in kibana sidebar from Open Distro for Elasticsearch (as dropdown) to Elasticsearch reporting
please guide me where I can change this.

**Comment by francisco-hoo** [comment]
Hey @rohitsawant02

To achieve that you will need to build the plugin. Changing the name on the sidebar menu just needs to change this line:
https://github.com/opendistro-for-elasticsearch/kibana-reports/blob/f013a40a3766dd261f806b72265d30265bd08ee8/kibana-reports/public/plugin.ts#L39

From `Reporting` to what you need.

You should take a look into this pull request (#362) that already offers you this through Translations.

**Comment by rohitsawant02** [comment]
Hi francisco-hoo,

Thanks for your response, I want to change the label 'Open Distro for Elasticsearch' to 'Elasticsearch reporting' but I am not able to build it properly. I am using branch opendistro-1.13.2.0 and building by starting kibana with command 'yarn start'. please guide us steps to change label and build the plugin properly.

**Comment by francisco-hoo** [comment]
Hey again @rohitsawant02,

You are talking about the Category label, for that you need change this:
https://github.com/opendistro-for-elasticsearch/kibana-reports/blob/f013a40a3766dd261f806b72265d30265bd08ee8/kibana-reports/public/plugin.ts#L42

If you're using another plugin from Open Distro, you'll need to change the same label in each plugin.

With `yarn start` you are starting a Kibana in Dev mode. If you try to change the code during the yarn start, you will see the Kibana server generate the new changes to be visible on the UI. After validating your changes, you can easily build the plugin by running `yarn build`.

Take a look on those steps:
https://github.com/opendistro-for-elasticsearch/kibana-reports/blob/f013a40a3766dd261f806b72265d30265bd08ee8/.github/workflows/kibana-reports-release-workflow.yml#L68

Here, it's possible to see what's needed to build the plugin with the right chromium version.

**Comment by rohitsawant02** [comment]
Hi @francisco-hoo ,

It is working now. thanks 
but I have another issue while downloading the report in pdf format that  Error: Protocol error (Page.printToPDF): Printing is not enabled . can you please help to resolve this issue.

**Comment by zhongnansu** [comment]
> Hi @francisco-hoo ,
> 
> It is working now. thanks
> but I have another issue while downloading the report in pdf format that Error: Protocol error (Page.printToPDF): Printing is not enabled . can you please help to resolve this issue.

I saw another [issue](https://github.com/opendistro-for-elasticsearch/kibana-reports/issues/375) created, closing this one. Also, I highly recommand you checkout our latest [repo](https://github.com/opensearch-project/dashboards-reports) in OpenSearch. This Open Distro project will not have major release in the future, we have migrated to OpenSearch and are slowly deprecating OpenDistro.
